app/src/main/java/com/asfoundation/wallet/ui/iab/AppCoinsPaymentMethod.kt

```diff
@@ -6,5 +6,7 @@ data class AppCoinsPaymentMethod(override val id: String, override val label: St
                                  val isAppcEnabled: Boolean = false,
                                  val isCreditsEnabled: Boolean = false, val appcLabel: String,
                                  val creditsLabel: String, val creditsIconUrl: String,
-                                 override var disabledReason: Int?) :
+                                 override var disabledReason: Int = -1,
+                                 val disabledReasonAppc: Int = -1,
+                                 val disabledReasonCredits: Int = -1) :
```

Comment:
We should keep the nullability I think, I think in kotlin it's easier to handle it and know when we should check for it than by using -1